_data-prepper/pipelines/configuration/processors/write_json.md

```diff
@@ -13,6 +13,6 @@ The `write_json` processor converts an object in an event into a JSON string. Yo
 
 | Option | Description | Example |
 | :--- | :--- | :--- |
-| source | Mandatory field that specifies the name of the field in the event containing the message or object to be parsed. | If `source` is set to `"message"` and the input is `{"message": {"key1":"value1", "key2":{"key3":"value3"}}`, then the `write_json` processor generates `{"message": "{\"key1\":\"value`\", \"key2\":"{\"key3\":\"value3\"}"}"`.
-| target | An optional field that specifies the name of the field in which the resulting JSON string should be stored. If `target` is not specified, then the `source` field is used.
+| source | Mandatory field that specifies the name of the field in the event containing the message or object to be parsed. | If `source` is set to `"message"` and the input is `{"message": {"key1":"value1", "key2":{"key3":"value3"}}}`, then the `write_json` processor generates `{"message": "{\"key1\":\"value\", \"key2\":{\"key3\":\"value3\"}}"}`.
```

Comment:
I just noticed two small errors. This should be:

```
"{\"key1\":\"value1\",\"key2\":{\"key3\":\"value3\"}}"
```

Reply:
To clarify, the part that is generated should be:

```
"{\"key1\":\"value1\",\"key2\":{\"key3\":\"value3\"}}"
```

You have a `value` instead of `value1` and also a space. The string will not have any spaces, and this may be a significant change.